ext/DFTKPlotsExt.jl

```diff
@@ -108,4 +108,98 @@ function plot_dos(basis, eigenvalues; εF=nothing, unit=u"hartree",
 end
 plot_dos(scfres; kwargs...) = plot_dos(scfres.basis, scfres.eigenvalues; scfres.εF, kwargs...)
 
+function plot_ldos(basis, eigenvalues, ψ; εF=nothing, unit=u"hartree",
```

Comment:
Is this plot easy to read. I mean it is a 3D thing mapped onto a single axis, indexed in the real-space grid, right ? Would a heatmap or something like this not be better ?